tests/test_homo_sapiens.py

```diff
@@ -109,11 +109,13 @@ class TestJacobsPapuansOutOfAfrica(
     qc_model = homo_sapiens_qc.DenisovanAncestryInPapuans()
 
 
-# Models that have not been QC'd:
-
 class TestGutenkunstThreePopOutOfAfrica(
         unittest.TestCase, test_models.CatalogDemographicModelTestMixin):
     model = species.get_demographic_model("OutOfAfrica_3G09")
```

Comment:
Same issue here as for the ZigZag model, regarding the need to instead inherit from `QcdCatalogDemographicModelTestMixin`.